gradle-jdks/src/test/resources/amazon-cas-with-whitespace-between.pem

```diff
@@ -0,0 +1,57 @@
+https://www.amazontrust.com/repository/SFSRootCAG2.pem
```

Comment:
didn't realise you could put non cert text in a pem file

Reply:
I am a firm believer that more people should annotate what the certs actually are so you can quickly browse it

Reply:
yeah definitely, that's amazing